socialhome/streams/app/tests/main.tests.js

```diff
@@ -1,7 +1,13 @@
 import "chai/register-should"
+import "streams/app/main"
+
+import Vue from "vue"
+
 
 describe("Test", () => {
```

Comment:
This describe should probably be called something else. Maybe `Main`? Since we're testing what `main.js` does.